samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md

```diff
@@ -43,21 +43,38 @@ For the purposes of this demonstration, all resources will be created in the us-
 2. In the Kubeflow Pipelines UI, upload this compiled pipeline specification (the *.tar.gz* file) and click on create run.
 3. Provide the sagemaker execution `role_arn` you created and `bucket_name` you created as pipeline inputs.
 4. Once the pipeline completes, you can go to `batch_transform_output` to check your batch prediction results.
-You will also have an model endpoint in service. Refer to [Prediction section](#Prediction) below to run predictions aganist your deployed model aganist the endpoint. Please remember to [clean up the endpoint](https://docs.aws.amazon.com/sagemaker/latest/dg/realtime-endpoints-delete-resources.html).
-
+You will also have an model endpoint in service. Refer to [Prediction section](#Prediction) below to run predictions aganist your deployed model aganist the endpoint.
 
 ## Prediction
 
-1. Find your endpoint name either by:
-    - Opening SageMaker [console](https://us-east-1.console.aws.amazon.com/sagemaker/home?region=us-east-1#/endpoints),  or
-    - Clicking the `sagemaker-deploy-model-endpoint_name` under `Output artifacts` of `SageMaker - Deploy Model` component of the pipeline run
-
+1. Find your endpoint name by:
+    - Checking the `sagemaker_resource_name` output field of the Endpoint component.
+    ```
+      export ENDPOINT_NAME=
+    ```
 2. Setup AWS credentials with `sagemaker:InvokeEndpoint` access. [Sample commands](https://sagemaker.readthedocs.io/en/v1.60.2/kubernetes/using_amazon_sagemaker_components.html#configure-permissions-to-run-predictions)
 4. Run the script below to invoke the endpoint
   ```
-    python invoke_endpoint.py <ENDPOINT-NAME>
+    python invoke_endpoint.py $ENDPOINT_NAME
   ```
 
+## Cleaning up the endpoint
+
+You can find the model/endpoint configuration name in the `sagemaker_resource_name` output field of the respective component.
+
+```
+export ENDPOINT_CONFIG_NAME=
+export MODEL_NAME=
+```
+To delete all the endpoint resources use:
+```
+export MY_KUBEFLOW_NAMESPACE=
```

Comment:
mention info that if using standalone, namespace is Kubeflow, if full kubeflow installation, find namespace at the top bar of the window

Reply:
done